Tweak/Core/Views/KayokoFavoritesTableView.m

```diff
@@ -27,7 +36,8 @@ - (UISwipeActionsConfiguration *)tableView:(UITableView *)tableView leadingSwipe
             completionHandler(YES);
         }];
         [linkAction setImage:[UIImage systemImageNamed:@"arrow.up"]];
-        [linkAction setBackgroundColor:[UIColor systemGreenColor]];
+        // rgba(39, 174, 96, 1.0)
+        baseColorSeries = [UIColor colorWithRed:39.0/255.0 green:174.0/255.0 blue:96.0/255.0 alpha:1.0];
```

Comment:
`colorWithRed:39.0/255.0 green:174.0/255.0 blue:96.0/255.0 alpha:1.0` is a mess. Use `colorWithRed:39 / 255 green:174 / 255 blue:96 / 255 alpha:1` instead.

Reply:
Well, `.0` means it's a **double**, not an **int**.